discord-scripts/discord-webhook.ts

```diff
@@ -97,5 +154,37 @@ export default async function webhookDiscord(
       },
     )
     robot.logger.info("Webhook is now enabled")
+
+    robot.router.post("/start-date", handleAuth, async (req, res) => {
```

Comment:
We should look at scoping this very specifically and making it a little more REST-like. Something like:
```
robot.router.post(`${webhookUrl}/user/:username/out-of-office`...

robot.router.delete(`${webhookUrl}/user/:username/out-of-office`...
```

Reply:
(I say this not remembering what `webhookUrl` is. We should also probably scope the message poster webhook to `/message` or similar.)